jabgui/src/main/java/org/jabref/gui/fieldeditors/identifier/IdentifierEditor.java

```diff
@@ -71,21 +71,19 @@ public IdentifierEditor(Field field,
         }
 
         ViewLoader.view(this)
-                  .root(this)
-                  .load();
+                .root(this)
+                .load();
```

Comment:
```suggestion
                 .root(this)
                 .load();
```
please re-add the space